src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -875,12 +875,6 @@ public override BoundNode VisitUsingStatement(BoundUsingStatement node)
                 return null;
             }
 
-            public override BoundNode VisitAwaitableValuePlaceholder(BoundAwaitableValuePlaceholder node)
```

Comment:
>VisitAwaitableValuePlaceholder [](start = 38, length = 30)

Can we keep this?

Reply:
Never mind, I see that we remove BoundAwaitableValuePlaceholder nodes later in lowering (ie. after local rewriting).

---
In reply to: [330247716](https://github.com/dotnet/roslyn/pull/38960#discussion_r330247716) [](ancestors = 330247716)